app-i18n/librime/librime-1.9.0.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 2012-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+inherit cmake multiprocessing
+
+DESCRIPTION="RIME (Rime Input Method Engine) core library"
+HOMEPAGE="https://rime.im/ https://github.com/rime/librime"
+SRC_URI="https://github.com/rime/librime/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD BSD-2 Boost-1.0 MIT"
+SLOT="0/1-${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-cpp/glog:=
+	>=dev-libs/boost-1.74:=
+	app-i18n/opencc:0=
+	dev-cpp/yaml-cpp:0=
+	dev-libs/leveldb:0=
+	dev-libs/marisa:0=
+"
+DEPEND="${RDEPEND}
+	test? ( dev-cpp/gtest )
+"
+
+DOCS=(CHANGELOG.md README.md)
+
+src_configure() {
+	local -x CXXFLAGS="${CXXFLAGS}"
```

Comment:
Any reason to keep this?
If it's for `CXXFLAGS+=" "`,  use `append-flags` or `append-cxxflags` from flag-o-matic.eclass

Reply:
Just keep it simple, there is no good reason to inherit an extra eclass for a temporary variable.